client/src/components/general/Layout.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import { Outlet } from "react-router-dom";
+import { Grid, Box } from "@mui/material";
+import Navbar from "../navigation/Navbar";
+import Sidebar from "../navigation/Sidebar";
+import ScrollTopBtn from "../navigation/ScrollTopBtn";
+
+/**
+ * This component is responsible for rendering the app.
+ * It displays the navbar on top and the sidebar on the left.
+ * The main content is rendered in the middle.
+ * @returns the app components
+ */
+export default function LayoutTest() {
```

Comment:
This component has a different name then the file. I would suggest the name "Layout".

Additionally: We use the arrow function format for functions (() => {...}).


```suggestion
export default function Layout = () => {
```